q2_sample_classifier/classify.py

```diff
@@ -175,8 +175,10 @@ def classify_samples(ctx,
     accuracy_results, = confusion(predictions, metadata, probabilities,
                                   missing_samples='ignore', palette=palette)
 
+    _heatmap, _ = heatmap(ctx, table, importance)
```

Comment:
maybe add `metadata=metadata, group_samples=True` and see what you think. This will speed things up by making a much smaller heatmap.

I think grouping could be justified if we consider this heatmap as a first-pass diagnostic... ultimately, users will probably want to build a new heatmap that is tailored more to their specifications.

Reply:
`metadata=metadata`  causes the test to fail 
![image](https://user-images.githubusercontent.com/10642616/62320149-5fdbbe80-b454-11e9-8ccc-7fdab3ad10e2.png)